libraries/core_libs/consensus/src/pbft/pbft_manager.cpp

```diff
@@ -1561,9 +1561,10 @@ bool PbftManager::pushCertVotedPbftBlockIntoChain_(taraxa::blk_hash_t const &cer
     return false;
   }
 
-  cert_sync_block_.cert_votes = std::move(cert_votes_for_round);
+  // TODO: Only include reward votes that are part of the pbft block and not all cert votes
+  cert_sync_block_.cert_votes = db_->getLastBlockCertVotes();
 
-  if (!pushPbftBlock_(std::move(cert_sync_block_), std::move(dag_blocks_order))) {
+  if (!pushPbftBlock_(std::move(cert_sync_block_), std::move(cert_votes_for_round), std::move(dag_blocks_order))) {
```

Comment:
current_round_cert_votes might be more appropriate ?

Reply:
Done